README.md

```diff
@@ -1,3 +1,5 @@
+[![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
+
```

Comment:
what is it for? :)

Reply:
To show that this repository is Commitizen friendly. Ie. our commits are clean ;)